.github/workflows/gke.yaml

```diff
@@ -33,34 +50,48 @@ jobs:
             echo ::set-output name=tag::${{ github.sha }}
           fi
 
+      - name: Set status check to pending
+        if: ${{ always() }}
+        uses: Sibz/github-status-action@v1.1.4
+        with:
+          authToken: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ steps.vars.outputs.tag }}
+          context: ${{ env.check_name }}
+          description: Connectivity test in progress...
+          state: pending
+          target_url: ${{ env.check_url }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Why are we doing `checkout` of the source code? Is the source code compiled anywhere?

Reply:
Leftover, removed :)